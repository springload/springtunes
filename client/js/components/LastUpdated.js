import React, { PropTypes } from 'react';

const propTypes = {
    lastUpdated: PropTypes.number,
};

const LastUpdated = ({ lastUpdated }) => (
    <div className='last-updated__time'>
        <p>
            Last updated at { new Date(lastUpdated).toLocaleTimeString() }
        </p>
    </div>
);

LastUpdated.propTypes = propTypes;

export default LastUpdated;
