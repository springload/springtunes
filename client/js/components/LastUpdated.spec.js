import React from 'react';
import renderer from 'react-test-renderer';
import LastUpdated from './LastUpdated';

describe('LastUpdated component', () => {
    it('renders correctly with min params', () => {
        const result = renderer
            .create(<LastUpdated lastUpdated={2342342342} />)
            .toJSON();
        expect(result).toMatchSnapshot();
    });
});
