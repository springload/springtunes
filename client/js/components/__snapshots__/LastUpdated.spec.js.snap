exports[`LastUpdated component renders correctly with min params 1`] = `
<div
  className="last-updated__time">
  <p>
    Last updated at 
    2:39:02 PM
  </p>
</div>
`;
